packages/fxa-auth-client/lib/client.ts

```diff
@@ -2244,6 +2260,54 @@ export default class AuthClient {
     );
   }
 
+  async recoveryPhoneCreate(
+    sessionToken: string,
+    phoneNumber: string,
+    headers?: Headers
+  ) {
+    return this.sessionPost(
+      '/recovery-phone/create',
+      sessionToken,
+      { phoneNumber },
+      headers
+    );
+  }
+
+  async recoveryPhoneAvailable(sessionToken: string, headers?: Headers) {
+    return this.sessionPost(
+      '/recovery-phone/available',
+      sessionToken,
+      {},
+      headers
+    );
+  }
+
+  async recoveryPhoneConfirm(
+    sessionToken: string,
+    code: string,
+    headers?: Headers
+  ) {
+    return this.sessionPost(
+      '/recovery-phone/available',
+      sessionToken,
+      { code },
+      headers
+    );
+  }
+
+  async recoveryPhoneSendCode(sessionToken: string, headers?: Headers) {
+    return this.sessionPost(
+      '/recovery-phone/send_code',
+      sessionToken,
+      {},
+      headers
+    );
+  }
+
+  async deleteRecoveryPhone(sessionToken: string, headers?: Headers) {
```

Comment:
nit: the other functions start with 'recoveryPhone' but this starts with the verb.

Reply:
Good point, I'll fix this in the test PR I'm currently working.